CIP-0001/README.md

```diff
@@ -120,12 +122,18 @@ Authors:
     - Duncan Coutts <duncan.coutts@iohk.io>
 Implementors: N/A
 Discussions:
-    - https://github.com/cardano-foundation/cips/pulls/1
+    - https://github.com/cardano-foundation/cips/pull/366
 Created: 2020-03-21
 License: CC-BY-4.0
 ---
 ```
 
+Especially because Markdown link syntax is not supported in the header preamble, labels can be added to clarify list items; e.g.:
+```
+Implementors:
```

Comment:
```suggestion
Discussions:
 - Original PR: https://github.com/cardano-foundation/CIPs/pull/366
```
I feel like this works as a better example, as `Implementors` can be referenced by their website address

Reply:
I agree with you, but wouldn't the YAML format label need double quotes around it? (https://docs.nobl9.com/yaml-guide)  If so, does this look OK?  And it would it render with or without quotes in the GitHub header?